recipes/find-kedro/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "find-kedro" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6d1d6c9e7baf5ddf190c1ff34b5a95a5f3eea2cdd0f8999fe452bb0dfea8f460
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+  entry_points:
+    - find-kedro = find_kedro.cli:cli
+
+requirements:
+  host:
+    - python
+    - pip
+    - kedro
+    - click
+    - colorama
+    - pygments
```

Comment:
Here you just need ``python`` and ``pip``
```suggestion
```

Reply:
Updated, thanks @marcelotrevisani